src/ResourceManager/Relay/Commands.Relay/Cmdlets/HybridConnections/NewAzureRelayHybridConnectionsAuthorizationRules.cs

```diff
@@ -0,0 +1,78 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Management.Automation;
+using Microsoft.Azure.Commands.Relay.Models;
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.Commands.Relay.Commands.HybridConnections
+{
+    /// <summary>
+    /// 'New-AzureRmRelayHybridConnectionAuthorizationRule' Cmdlet creates a new AuthorizationRule
+    /// </summary>
+    [Cmdlet(VerbsCommon.New, RelayHybridConnectionAuthorizationRuleVerb, SupportsShouldProcess = true), OutputType(typeof(AuthorizationRuleAttributes))]
+    public class NewAzureRelayHybridConnectionsAuthorizationRule : AzureRelayCmdletBase
```

Comment:
Why is this cmdlet still here?  This should be replaced by `New-AzureRmRelayAuthorizationRule``` cmdlet